CMakeLists.txt

```diff
@@ -84,6 +87,7 @@ if (CATKIN_ENABLE_TESTING)
   target_link_libraries(test_subclass ${catkin_LIBRARIES} ${PROJECT_NAME}_solver joint_state_listener)
 
   install(FILES test/one_link.urdf test/pr2.urdf test/two_links_fixed_joint.urdf test/two_links_moving_joint.urdf DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/test)
+  install(FILES test/one_link.urdf test/pr2.urdf test/two_links_fixed_joint.urdf test/two_links_moving_joint.urdf test/frames_and_slashes.urdf DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/test)
```

Comment:
This line is redundant, isn't it?  That is, shouldn't you just be able to add `test/frames_and_slashes.urdf` to the preceding line and be done with it?

Reply:
This comment is still open.

Reply:
Rebased and removed redundant install in e817dcf